contrib/babelfishpg_tsql/src/catalog.c

```diff
@@ -344,14 +346,25 @@ babelfish_helpdb(PG_FUNCTION_ARGS)
 	if (PG_NARGS() > 0)
 	{
 		dbname = TextDatumGetCString(PG_GETARG_DATUM(0));
-		if (!DbidIsValid(get_db_id(dbname)))
+		dbname_lower = str_tolower(dbname, strlen(dbname), DEFAULT_COLLATION_OID);
+		/* Remove trailing spaces at the end of user typed dbname */
+		index = -1;
+		for (int i = 0; dbname_lower[i]!='\0'; i++)
+		{
+			if (dbname_lower[i]!=' ')
+			{
+				index = i;
+			}
+		}
```

Comment:
nitpick
(1) is there already a trim() function in the code base we can use?
(2) do we need to pfree dbname_lower?